apps/dh/api-dh/source/DataHub.WebApi/Modules/Charges/Extensions/ChargeExtensions.cs

```diff
@@ -19,31 +19,30 @@ namespace Energinet.DataHub.WebApi.Modules.Charges.Extensions;
 
 public static class ChargeExtensions
 {
-    public static ChargeInformationPeriodDto? GetCurrentPeriod(this ChargeInformationDto charge)
-    {
-        return charge.Periods
+    public static ChargeInformationPeriodDto? GetCurrentPeriod(this ChargeInformationDto charge) => charge.Periods
```

Comment:
question: should we still use `currentPeriod` in the frontend (for things like description, name etc). Or should that use `latestPeriod` instead? How do we know when to use what? 😄

Reply:
Yes, we should change that. I will make a PR for it :-)